tests/compare_mozilla.rs

```diff
@@ -140,6 +142,8 @@ fn test_contains_most_roots_known_by_mozilla() {
 
 #[test]
 fn util_list_certs() {
+    common::clear_env();
```

Comment:
This test is not annotated with `#[serial]` but it does use `clear_env()` which AIUI is undefined behavior:

> In multi-threaded programs, you must ensure that are no other threads concurrently writing or reading(!) from the environment through functions other than the ones in this module. You are responsible for figuring out how to achieve this, but we strongly suggest not using set_var or remove_var in multi-threaded programs at all.

We should probably document this on `clear_env()`?

Reply:
Will do.

I guess that's for another time but I wonder if I should look into adding a `serial_test::is_serial()` for checking if we are in a `#[serial]` context. Then, we could just add an assert in `clear_env()`.

Reply:
I ended up marking clear_env() unsafe. It's up to the caller to ensure safety, so it really should be marked this way, I think.